examples/minimal/outputs.tf

```
@@ -0,0 +1,52 @@
+output "s3_bucket_arn" {
+  value       = module.mwaa.s3_bucket_arn
+  description = "ID of S3 bucket."
```

```suggestion
  description = "ARN of S3 bucket."
```

do we also need to output the bucket ID/name?